webapp-django/crashstats/crashstats/static/crashstats/js/socorro/oauth2.js

```diff
@@ -58,6 +58,17 @@ var OAuth2 = (function() {
         });
     };
 
+    /* http://stackoverflow.com/a/901144/205832 */
+    var getParameterByName = function getParameterByName(name, url) {
+        if (!url) url = window.location.href;
+        name = name.replace(/[\[\]]/g, "\\$&");
+        var regex = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)"),
+            results = regex.exec(url);
+        if (!results) return null;
+        if (!results[2]) return '';
+        return decodeURIComponent(results[2].replace(/\+/g, " "));
+    };
```

Comment:
I know it comes from stackoverflow and you didn't write it, but I would still make this pass our linter. Notably, no inlined ``if`` statement and not multiple ``var`` declarations.